deploy/kubernetes/dev/controller.yaml

```diff
@@ -17,10 +17,10 @@ spec:
       containers:
         - name: csi-provisioner
           imagePullPolicy: Always
-          image: quay.io/k8scsi/csi-provisioner:v0.2.0
+          image: quay.io/k8scsi/csi-provisioner:v0.3.1
           args:
             - "--v=5"
-            - "--provisioner=csi-gce-pd"
+            - "--provisioner=com.google.csi.gcepd"
```

Comment:
you don't need to set provisioner name

Reply:
https://github.com/kubernetes-csi/external-provisioner/pull/109#issuecomment-425280588